src/main/java/witchinggadgets/client/render/TileRenderWallMirror.java

```diff
@@ -53,8 +53,8 @@ public void renderTileEntityAt(TileEntityWallMirror tile, double x, double y, do
 
         Tessellator tes = Tessellator.instance;
 
-        // GL11.glEnable(3042);
-        // GL11.glBlendFunc(770, 771);
+        // GL11.glEnable(GL11.GL_BLEND);
```

Comment:
Delete this?

Reply:
I also didn't touch that, applyDecompilerCleanupToMain did but sure I can go back and delete it